### ISSUE TITLE
configure script problem

### ISSUE THREAD

**Issue by VictorEijkhout** [NanoComp/meep#1819]
Configure gives the following error:

```
checking how to activate readline in Guile... ice-9 readline
checking for libctl dir... ./configure: line 29499: $'\357\277\274': command not found
configure: error: could not find libctl files; use --with-libctl=<dir>
```

which I can fix as follows:

```
if test x != x"$LIBCTL_DIR" -a -r "$LIBCTL_DIR/share/libctl/base/ctl.scm"; then
##￼     LIBCTL_DIR="$LIBCTL_DIR/share/libctl"
    LIBCTL_DIR="${LIBCTL_DIR}/share/libctl"
fi
```

But I don't understand why this works, or for that matter where the error comes from to begin with.

**Comment by VictorEijkhout** [comment]
Oh wait, you have some non-ascii white space in that line.

**Comment by oskooi** [comment]
Please note that GitHub issues are for filing bug reports and feature requests. They are not a general help desk. 

It seems the issues you are running into are related to installing software on Unix-like systems and not specific to Meep.

**Comment by VictorEijkhout** [comment]
My problems are specifically with the meep installer. Can you confirm that your own configure script does not have junk on that line?

**Comment by lockywolf** [comment]
meep has a problem with  the `./configure` script, but if you re-generate one with `./autogen.sh`

According to GNU, this is a wrong behaviour, the configure script should be portable. But almost every system now has autotools, so it should not be _much_ of a problem to re-generate one.

**Comment by oskooi** [comment]
Fixed by #2273.
